ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/TaskList/Decision/AnyConditionsIntegrationTests.cs

```diff
@@ -8,93 +9,95 @@
-			await Document.QuerySelector<IHtmlButtonElement>("#submit-btn").SubmitAsync();
+			await _wizard.SetIsConditionalAndContinue(true, details);
 
-			await OpenUrlAsync($"/task-list/{project.Id}/decision/any-conditions");
+			await OpenUrlAsync($"/task-list/{_project.Id}/decision/any-conditions");
 
-			var formElement = Document.QuerySelector<IHtmlInputElement>("#no-radio");
+			bool isConditional = Document.QuerySelector<IHtmlInputElement>("#yes-radio").IsChecked;
+			string conditionDetails = Document.QuerySelector<IHtmlTextAreaElement>("#ApprovedConditionsDetails")?.Text().Trim();
 
-			formElement.IsChecked.Should().BeTrue();
+			isConditional.Should().BeTrue();
+			conditionDetails.Should().Be(details);
 		}
 
 		[Fact]
-		public async Task Submit_yes_should_redirect_to_what_conditions()
+		public async Task Submit_yes_should_redirect_to_decision_date()
 		{
-			var project = AddGetProject(p => p.GeneralInformationSectionComplete = false);
-
-			await OpenUrlAsync($"/task-list/{project.Id}/decision/any-conditions");
+			await _wizard.SetIsConditionalAndContinue(true, "Reasons");
 
-			Document.QuerySelector<IHtmlInputElement>("#yes-radio").IsChecked = true;
-			await Document.QuerySelector<IHtmlButtonElement>("#submit-btn").SubmitAsync();
-
-			Document.Url.Should().EndWith("/decision/what-conditions");
+			Document.Url.Should().EndWith("/decision/decision-date");
 		}
 
 		[Fact]
 		public async Task Submit_no_should_redirect_to_decision_date()
 		{
-			var project = AddGetProject(p => p.GeneralInformationSectionComplete = false);
-
-			RecordDecisionWizard wizard = new RecordDecisionWizard(Context);
-
-			await wizard.StartFor(project.Id);
-			await wizard.SetDecisionToAndContinue(AdvisoryBoardDecisions.Approved);
-			await wizard.SetDecisionByAndContinue(DecisionMadeBy.Minister);
-			await wizard.SetIsConditionalAndContinue(false);
+			await _wizard.SetIsConditionalAndContinue(false, default);
 
 			Document.Url.Should().EndWith("/decision/decision-date");
 		}
 
 		[Fact]
-		public async Task Should_go_back_to_whodecided()
+		public async Task Should_go_back_to_who_decided()
 		{
-			var project = AddGetProject(p => p.GeneralInformationSectionComplete = false);
-
-			await OpenUrlAsync($"/task-list/{project.Id}/decision/any-conditions");
-
 			await NavigateAsync("Back");
 
-			Document.QuerySelector<IHtmlElement>("h1").Text().Trim().Should().Be("Who made this decision?");
+			PageHeading.Should().Be("Who made this decision?");
 		}
 
 		[Fact]
 		public async Task Should_display_error_when_nothing_selected()
 		{
-			var project = AddGetProject(p => p.GeneralInformationSectionComplete = false);
+			await _wizard.ClickSubmitButton();
+
+			Document.QuerySelector<IHtmlElement>("[href='#ApprovedConditionsSet']")?.Text()
+				.Should().Be("Select whether any conditions were set");
+
+			PageHeading.Should().Be("Were any conditions set?");
+		}
+
+		#region IAsyncLifetime implementation
+
+		public async Task InitializeAsync()
```

Comment:
Please can we put this at the top, just so its clearer what is happening before each test.

Reply:
Done - was at the bottom as Resharper throws regions down there. I'd put it in a region to keep them together as I _thought_ they'd be separated when sorted during code reformatting - but actually weren't in the end :)